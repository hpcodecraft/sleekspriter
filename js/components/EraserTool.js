var EraserTool = React.createClass({
  render: function() {
    return (
      <div id="Eraser-Tool" className="ToolComponent">
        <i className="fa fa-eraser"></i>

        <span className="hint">Click a pixel to erase it.</span>
      </div>
    );
  }
});