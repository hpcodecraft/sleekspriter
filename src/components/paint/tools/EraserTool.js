import React from "react";
import { t } from "../../../utils";

const EraserTool = (props) => {
  return (
    <div id="Eraser-Tool" className="ToolComponent">
      <i className="icon flaticon-double31" style={{position:"relative", left: "0.25em"}}></i>
      <span className="hint">{t("Click a pixel to erase it.")}</span>
    </div>
  );
};

export default EraserTool;
