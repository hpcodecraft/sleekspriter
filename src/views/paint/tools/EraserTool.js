import React from 'react';

const EraserTool = (props) => {
  return (
    <div id="Eraser-Tool" className="ToolComponent">
      <i className="icon flaticon-double31" style={{position:'relative', left: '0.25em'}}></i>
      <span className="hint">Click a pixel to erase it.</span>
    </div>
  );
};

export default EraserTool;
