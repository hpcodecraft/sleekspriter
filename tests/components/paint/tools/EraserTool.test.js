import EraserTool from "components/paint/tools/EraserTool";

describe("EraserTool", () => {
  it("should render correctly", () => {
    const component = shallow(<EraserTool />);
    expect(component.html()).toMatchSnapshot();
  });
});
