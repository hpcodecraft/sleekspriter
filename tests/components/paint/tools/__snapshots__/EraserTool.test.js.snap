// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EraserTool should render correctly 1`] = `"<div id=\\"Eraser-Tool\\" class=\\"ToolComponent\\"><i class=\\"icon flaticon-double31\\" style=\\"position:relative;left:0.25em\\"></i><span class=\\"hint\\">Click a pixel to erase it.</span></div>"`;
